app-catalog/src/components/charts/EditorDialog.tsx

```diff
@@ -53,8 +55,22 @@ export function EditorDialog(props: {
     }
   }, [selectedNamespace, namespaceNames]);
 
+  // Fetch chart values for a given package and version
+  function refreshChartValue(packageID: string, packageVersion: string) {
+    fetchChartValues(packageID, packageVersion)
+      .then((response: any) => {
+        setChartValues(response);
+        setDefaultChartValues(yamlToJSON(response));
+      })
+      .catch(error => {
+        enqueueSnackbar(`Error fetching chart values ${error}`, {
+          variant: 'error',
+        });
+      });
+  }
+
   function handleChartValueFetch(chart: any) {
-    const packageID = chart.package_id;
+    const packageID = globalThis.CHART_PROFILE === VANILLA_HELM_REPO ? chart.name : chart.package_id;
```

Comment:
Instead of using global variables for this, can you please pass a prop into EditorDialog?